topology/sof/pipe-low-latency-playback.m4

```diff
@@ -32,21 +32,21 @@ include(`mixercontrol.m4')
 #
 # Volume Mixer control with max value of 32
 C_CONTROLMIXER(PCM PCM_ID Playback Volume, PIPELINE_ID,
-	CONTROLMIXER_OPS(volsw, 256 binds the mixer control to volume get/put handlers, 256, 256),
+	CONTROLMIXER_OPS(volsw, 1 binds the mixer control to volume get/put handlers, 1, 1),
```

Comment:
same here, not sure why you are changing this binding.

Reply:
see my above comments and thesofproject/linux#39